backend/bindings.py

```
@@ -24,14 +24,75 @@ def get_lib_path():
+
+        def start_udp_listener(self, port, callback):
+            # callback is expected to be a ctypes CFUNCTYPE or callable taking (message, sender_ip)
+            if self._running:
+                return
+            self._running = True
+            self._sock = _socket.socket(_socket.AF_INET, _socket.SOCK_DGRAM)
+            try:
+                self._sock.bind(("", port))
+            except Exception:
+                # fallback bind may fail in some environments; ignore
+                pass
```

Ruff (0.14.0)

`try`-`except`-`pass` detected, consider logging the exception

(S110)